### ISSUE TITLE
💡[Feature]: Sarcasm Detection in Cross Domain Applications

### ISSUE THREAD

**Issue by ShraddhaSabde** [recodehive/machine-learning-repos#691]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Sarcasm Detection in Cross Domain Applications
This project proposes the accuracy and efficiency of ML and NN models trained on one dataset and tested on other dataset. SARC dataset is used for training and amazon review dataset is used for testing the models. This enables Sarcasm detection on Cross Domain applications.

### Use Case

System Requirements
1. Python 2.7
 2. Python package Gensim, NLTK, Keras,Matplotlib, Numpy, SkLearn, Pandas, Re, Tensorflow 2.0

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by ShraddhaSabde** [comment]
Sir please Ignore above commits it is taking my older repositories commits also.

**Comment by ShraddhaSabde** [comment]
Sir can you please help 
I want to pull a request but, while checking commits it is also taking old local commits which I don't want to consider there are total 8 unwanted commits coming, which increases file size I want to remove that what can I do 
I have checked all resources to do that but not removing that commits.
can you tell how to do that on github?
![commits](https://github.com/user-attachments/assets/8851b7d1-927a-42fe-b1c8-22bc2c9118ab)

**Comment by sanjay-kv** [comment]
I will pass to the community members so they can help you

**Comment by ShashwatAgrawal20** [comment]
@ShraddhaSabde are you still looking for a solution?

**Comment by ShraddhaSabde** [comment]
ok sir Thank You!

**Comment by ShraddhaSabde** [comment]
Thank You sir Changes done
